google/ggrc-core
# GGRC-6540 JS error while opening a control page

# Issue description

Issue is similar #8724. On some Controls pages error appears: `Uncaught TypeError: Cannot read property 'id' of undefined`

# Steps to test the changes

- Create Control
- Create Proposal
- Create Workflow
- Map control to Task Group
- Create Proposal

After applying first .js commit error should disappear, but `task_group_objects` Attributes in Change Proposals should be shown.
After migration and second commit no `task_group_objects` should appear on Change Proposals tab.

# Solution description

BE solution is similar to  #8724

# Sanity checklist




# PR Review checklist
